### PR TITLE
Map Fix

### PR DESCRIPTION
**Pull request by Pyko1**
Fixed accidental space tile in insanity ward.
Fixed Isolation APC placement.
Fixed Isolation Airalarm placement.

### CODE REVIEW

**Comment by Pyko1** [comment]
Minor area fix at sec. 
Faded photo removed from z4, was causing runtimes according to carbonhell.

**Comment by Pyko1** [comment]
the space tile fix is quite urgent, and all these are fixes. Would be nice if we could get quick merge/compile.

File: tgstation.dme [hunk 1]
```diff
@@ -6,6 +6,77 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"
```
Comment (Carbonhell) [inline tgstation.dme]: Remove the file dirs

**Comment by Nebyab** [comment]
yeah sure

**Pull request merged**